build_stats.sh

```diff
@@ -12,9 +12,15 @@ cd R-3.3.2
 ./configure --prefix=${INST_DIR} --enable-R-shlib
 make
 make install
-
 cd ..
 
+# Correct PATH and LD_LIBRARY_PATH variables.
+export PATH=$PATH:${INST_DIR}
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${INST_DIR}/lib/R/lib
```

Comment:
Other way around, or we could pick up a system installed R.

```
export PATH=${INST_DIR}:${PATH}
```

Same for LD_LIBRARY_PATH.

Also, don't forget braces on LD_LIBRARY_PATH for consistency.